.github/workflows/main.yml

```diff
@@ -16,9 +16,10 @@ on:
 
 env:
   IMAGE: frontstage
-  REGISTRY_HOSTNAME: eu.gcr.io
-  HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
-  RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
+  REGISTRY_HOSTNAME: europe-west2-docker.pkg.dev
```

Comment:
It can stay, but generally clean separate cards are better

Reply:
Changes have been merged into to parent branch so no need to include here now